app/graphql/types/extensions/solr_search.rb

```diff
@@ -35,11 +36,11 @@ def after_resolve(object:, arguments:, value:, context:, memo:)
       # parse the facet objects or return an empty array
       def facets(facet_counts)
         return [] unless facet_counts['facet_fields']
-        display_label_map = Hyacinth::DigitalObject::Facets.facet_display_label_map
+        # display_label_map = Hyacinth::DigitalObject::Facets.facet_display_label_map
         facet_counts['facet_fields'].map do |facet_field, value_counts|
           {
             field_name: facet_field,
-            display_label: display_label_map[facet_field] || facet_field.split('_')[0...-1].map(&:titlecase).join(' '),
+            display_label: facet_display_label_map[facet_field] || facet_field.split('_')[0...-1].map(&:titlecase).join(' '),
```

Comment:
Doesn't this end up invoking the `facet_display_label_map` method once per map iteration instead of caching the result and reusing it?

Reply:
😂 yes, yes it does. what can I say I love geometric performance problems

Reply:
I guess we should *not* delegate to the other class.

Reply:
👍